packages/connect-express/package.json

```diff
@@ -20,10 +20,6 @@
   "main": "./dist/cjs/index.js",
   "exports": {
     ".": {
-      "node": {
-        "import": "./dist/proxy/index.js",
-        "require": "./dist/cjs/index.js"
-      },
```

Comment:
I believe that we also need to remove the "module" condition. Otherwise, we regress on https://github.com/bufbuild/protobuf-es/issues/610#issuecomment-1798866779

I'm not sure whether removing the "module" condition will break other bundlers, so we have to test with https://github.com/connectrpc/examples-es/pull/1149 / https://github.com/connectrpc/examples-es/pull/1002 again 🙄

Reply:
OK, I'll test with those branches. Re: the above question -- I can't remember what framework it was. I know connect-playwright had issues with the `node` condition, but other than that and Nuxt, I can't remember another one.

Reply:
To clarify: We _need_ to remove the "module" condition, otherwise we regress on https://github.com/bufbuild/protobuf-es/issues/610#issuecomment-1798866779. If this puts us into a lot of trouble, we have to step back to the drawing board, but right now, let's remove the condition. Can you do that?